README.md

```diff
@@ -63,6 +63,16 @@ useMousePosition className =
     name = uniqueNameFromCurrentCallStack { skipFrames: 2 }
 ```
 
+### Continuation-Passing Style
+
+You can use CPS using the `withHook` or `withHookCurried` functions, or their respective operators (`==>` and `=/>`).
```

Comment:
I would rather not even mention `withHook` or `withHookCurried`, it might create a false impression that you're supposed to use them. Just mention the operators, and if somebody is really interested, they can always discover the alias target via Pursuit.

Also, I would also add motivation for why you might want to do this.

```suggestion
If you're only using a single hook, sometimes it might be more concise to use CPS via the `==>` or `=/>` operators.
```

Reply:
Good idea 👍🏻